File: backend/src/main/java/com/grupo6daw/lcdd_daw/dto/GameDTO.java
package com.grupo6daw.lcdd_daw.dto;

public record GameDTO(
        Long gameId,
        String gameName,
        String gameDescription,
        ImageDTO gameImage,
        Long minPlayers,
        Long maxPlayers,
        Long minDuration,
        Long maxDuration,
        String genre) {
}

File: frontend/app/dtos/GameDTO.ts
import type ImageDTO from "./ImageDTO";

export default interface GameDTO {
    gameId: number,
    gameName: string,
    gameDescription: string,
    gameImage: ImageDTO,
    minPlayers: number,
    maxPlayers: number,
    minDuration: number,
    maxDuration: number,
    genre: string
}